src/components/modal/modal.tsx

```diff
@@ -255,6 +264,22 @@ export class Modal
     ) : null;
   }
 
+  renderContentTop(): VNode {
+    return (
+      <div class={CSS.contentTop} hidden={!this.hasContentTop} key="content-top">
```

Comment:
It looks like the content top and bottom nodes are always rendered, so these don't need `key` (see [Stencil doc on conditional rendering](https://stenciljs.com/docs/templating-jsx#:~:text=If%20this%20array%20of%20children%20is%20dynamic%2C%20i.e.%2C%20if%20any%20nodes%20may%20be%20added%2C%20removed%2C%20or%20reordered%2C%20then%20it%27s%20a%20good%20idea%20to%20set%20a%20unique%20key%20attribute%20on%20each%20element)).